compose/cli/main.py

```diff
@@ -292,6 +292,13 @@ def run(self, project, options):
         """
         service = project.get_service(options['SERVICE'])
 
+        legacy.check_for_legacy_containers(
+            project.client,
+            project.name,
+            [service.name],
+            allow_one_off=False,
+        )
```

Comment:
Since this is not a one-time cost, it might be nice to provide a `--skip-legacy-check` for users that know their containers are gone, and don't want to incur the extra overhead of the "list all containers" api call.